### PR TITLE
Fix table name sanitation / enable table schemes

### PR DESCRIPTION
**Pull request by annando**
This is an alternative solution for PR https://github.com/friendica/friendica/pull/7174

It solves the problem that some database commands hadn't sanitized the table name. And it adds the functionality of being able to work with database schemes.

This is a low impact solution and it doesn't work for every command at the moment. This is done since we are now in RC phase where the changes shouldn't be severe.

### CODE REVIEW

**Comment by annando** [comment]
Thanks for the hint with the identifiers.

**Comment by MrPetovan** [comment]
This PR is correct but I still prefer mine for its completeness.

File: src/Database/DBA.php [hunk 1]
```diff
@@ -872,6 +885,29 @@ public static function fetch($stmt) {
 		return $columns;
 	}
 
+	/**
+	 * @brief Insert a row into a table
```
Comment (nupplaphil) [inline src/Database/DBA.php]: PHP-Doc isn't right for this function, is it?

**Comment by nupplaphil** [comment]
I'm moew with @MrPetovan , but I leave the decission up to you.

**Comment by MrPetovan** [comment]
You're what? 😳

**Comment by nupplaphil** [comment]
I'm moew :D .. This is a new kind of feeling.. (for real: it was "more")

**Comment by annando** [comment]
This PR is explicitly done this way since we are currently in the RC phase. We can be sure that it will work well and doesn't have any side effect. The other PR is much more complicated and inherits the great danger that it does have side effects, possibly at the cascaded deletion of table entries or something else that cannot be overseen easily. That is a PR for the develop branch that can be tested thoroughly in the next months.

**Comment by MrPetovan** [comment]
Fair enough, I'll be fixing the conflicts anyway.

**Pull request merged**